base_path: "/sys/fs/cgroup"
settings:
  - directory: "autoware/system_monitor"
    search_word: 
      - "__node:=system_monitor_container"